src/react/deckgl.js

```diff
@@ -91,6 +91,17 @@ export default class DeckGL extends React.Component {
     }
     this._updateLayers(this.props);
 
+    const hasEvent = this.props.onLayerClick !== noop || this.props.onLayerHover !== noop;
+    const hasPickableLayer = this.layerManager.layers.map(l => l.props.pickable).includes(true);
+
+    if (hasEvent && !hasPickableLayer) {
+      console.warn([ //eslint-disable-line
```

Comment:
Strongly recommend using `log.once` here. If someone creates multiple DeckGL component, or destroys and recreates DeckGL this will quickly become annoying. Grep the code for examples.